search/lib/application.js

```diff
@@ -8,13 +8,27 @@ const { errorHandler } = require('./error-handler');
 const { logger } = require('./util');
 const settings = require('./settings');
 
+/**
+ * Express middleware for rewriting URLs to allow for redirects in AWS.
+ */
+const urlRewriteMiddleware = (req, res, next) => {
+  const routeAliases = settings.cmrStacRouteAliases ? settings.cmrStacRouteAliases.split(',') : ['/cmr-stac'];
```

Comment:
We could use do `const routeAliases = settings.cmrStacRouteAliases.split(',');`, couldn't we?